tests/test_features/test_pca.py

```diff
@@ -64,39 +65,101 @@ def binary(request):
     request.cls.dataset = Dataset(X, y)
```

Comment:
Can we change this to `request.cls.binary` just to make it more obvious?

Reply:
Will do it!